src/videoframeprocessor/VideoFrameProcessorTimer.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * [[VideoFrameProcessorTimer]] defines a timer used in the context of video frame processing.
+ */
+export default interface VideoFrameProcessorTimer {
```

Comment:
may not be needed now, but should we make `start` and `destroy` promise based? We can make this interface destroyable by extending existing `Destroyable` interface:
https://github.com/aws/amazon-chime-sdk-js/blob/faa246235609d63486a1d20ed54026ff78f82e50/src/destroyable/Destroyable.ts#L11

Reply:
good point, will change! I wish js had a better way of transitioning from sync -> async APIs in a backwards compatible way.